xds/internal/xdsclient/client.go

```diff
@@ -24,13 +24,34 @@ import (
 	"fmt"
 	"time"
 
+	"google.golang.org/grpc"
 	"google.golang.org/grpc/internal/grpclog"
 	"google.golang.org/grpc/internal/grpcsync"
 	"google.golang.org/grpc/xds/internal/xdsclient/bootstrap"
 	"google.golang.org/grpc/xds/internal/xdsclient/pubsub"
 	"google.golang.org/grpc/xds/internal/xdsclient/xdsresource"
 )
 
+var controllerDialOpts []grpc.DialOption
```

Comment:
This and the exported functions should be in the `controller` package, right?
So they are `controller.SetXXX()`, not `xdsclient.SetControllerXXX()`